### PR TITLE
Pr/372

### PR DESCRIPTION
**Pull request by KostyaSha**
More reworked #372 

@marcuslinke for review

@marcuslinke we should define what to do with javadocs, annotating only field sounds logical, but getter/setter should also provide docs. Didn't found way inherit field javadoc to methods.
Probably it makes sense annotate field and provide link to it from getter WDYT?

### CODE REVIEW

**Comment by marcuslinke** [comment]
LGTM :+1:  Really appreciate yor work. However, do you really want to investigate the remote API version for each single property? This seems to be really much effort. For me personally this effort would be to much because of my limited time resources.

**Comment by KostyaSha** [comment]
@marcuslinke it very convenient when implementing something to rely on features. I will try sync existed and we should require javadocs for new PRs imho. Plus i usually digging into depth when using tools :D

**Comment by KostyaSha** [comment]
Let's experiment, i will keep docs under fields and link from getters. Should be easy move them later.

File: src/main/java/com/github/dockerjava/api/command/InspectContainerResponse.java [hunk 1]
```diff
@@ -225,49 +230,184 @@ public String toString() {
...
+        /**
+         * See {@link #finishedAt}
+         */
+        @CheckForNull
         public String getFinishedAt() {
             return finishedAt;
         }
 
         @Override
+        public boolean equals(Object o) {
```
Comment (KostyaSha) [inline src/main/java/com/github/dockerjava/api/command/InspectContainerResponse.java]: @marcuslinke please check, btw why nested classes are used? Can they be extracted into model objects?

Comment (marcuslinke) [inline src/main/java/com/github/dockerjava/api/command/InspectContainerResponse.java]: Nesting classes makes sense when the inner class is used only in the context of the surrounding class and not anywhere else. Beside this it also documents the classes relationship in my oppionion.

Comment (KostyaSha) [inline src/main/java/com/github/dockerjava/api/command/InspectContainerResponse.java]: But you will have only field with cleaner class instead.

Comment (KostyaSha) [inline src/main/java/com/github/dockerjava/api/command/InspectContainerResponse.java]: Keeping as is.

**Comment by KostyaSha** [comment]
No objections - merging.

**Comment by marcuslinke** [comment]
@KostyaSha Please add to CHANGELOG

**Comment by KostyaSha** [comment]
@marcuslinke i will sync milestone to changelog later

**Comment by marcuslinke** [comment]
:+1: Thanks!

**Pull request merged**